gatling-core/src/main/scala/io/gatling/core/json/Json.scala

```diff
@@ -45,7 +49,7 @@ object Json {
   }
 
   private def writeString(s: String, rootLevel: Boolean) = {
-    val escapedLineFeeds = s.replace("\n", "\\n")
+    val escapedLineFeeds = objectMapper.writeValueAsString(s).replaceAll("^\"|\"$", "")
```

Comment:
I absolutely agree. And I believe that Jackson got it right. We should fix the way gatling build JSON probably. I tried to keep the change contained. Please decide.